### DISCUSSION TITLE
Is this possible to redirect to specific url after uninstalling the application?

### DISCUSSION THREAD

**Discussion by nomandhoni-cs** [tauri-apps/tauri#11593] [Q&A]
Is this possible to redirect to specific url after uninstalling the application?

**Comment by FabianLars** [comment]
It's possible yes, but there's no built-in thing to do that. You'll have to modify the wix and/or nsis template(s) yourself (google should know what you need to do).
https://v2.tauri.app/distribute/windows-installer/#customizing-the-wix-installer
https://v2.tauri.app/distribute/windows-installer/#customizing-the-nsis-installer
